projects/components/src/utils/test/widget-object/cypress/cypress-widget-finder.ts

```diff
@@ -37,8 +38,8 @@ export class CypressWidgetObjectFinder<T> {
         findOptions?: FindCypressWidgetOptions
     ): W {
         let tagName = widgetConstructor.tagName;
-        if (findOptions?.cssSelector) {
-            tagName += `${findOptions.cssSelector}`;
+        if (findOptions) {
+            tagName += SelectorUtil.extractSelector(findOptions);
```

Comment:
I should have deleted these 4 lines of code. You should be able to remove them. `let tagName` is not used.

Reply:
I removed them.